activities/codebase-stewardship/assessment-issue-template.md

```diff
@@ -0,0 +1,26 @@
+---
+type: Resource
+---
+
+# Template for assessment issues
+
+The text below can be used as a template for creating an issue in a repository for codebase where we are [starting an assessment](open-assessment.md). For easiest copy-paste it is rendered with code style here.
+
+## Template text
+
+```
+**Purpose**
+
+This issue helps tracking the assessment of this repository on it's way to [stewardship](https://about.publiccode.net/activities/codebase-stewardship/) by the Foundation for Public Code. The assessment is compiled into this [spreadsheet](linkToSpreadsheet).
```

Comment:
I think Boris' suggestion above is good. 

I would also consider adding a double space after the first sentence to start

``` 
The assessment is compiled into this [spreadsheet](linkToSpreadsheet).
``` 
on a different line

Reply:
My thinking on leaving it in there that if you are forgetting the link, someone will notice that the link is obviously broken and complain, presenting an opportunity to correct it. If the text is made to work without a link and one forgets to add it, no one will notice and just take for granted that we the spreadsheet were never meant to be shared and not complain. This will make us work less in the open which is against the principles for the assessment.

Reply:
interesting.. so kinda like using technical debt as a leverage to enforce openness? :thinking:

Reply:
I wouldn't call it that. More of a fallback that actually works. If we are counting on people to make mistakes (which is the premise of this being discussed at all), how do we make it possible for people to actually notice and tell us?

Secondly, I don't think we should use double spaces a workaround for an actual line break. But also, I don't feel a line break here is needed. Perhaps a native speaker have viewpoints on this. @ElenaFdR ?

Reply:
I agree, don't use line breaks unless they have an actual semantic meaning (such as in addresses)

This was an interesting discovery for me as I'd always make copy pastable things in a way that minimises perceived mistakes as opposed to signaling made mistakes. But I agree, perhaps our mistakes should be as visible as possible.